ddtrace/llmobs/_experiment.py

```diff
@@ -145,12 +145,10 @@ def delete(self, index: int) -> None:
         del self._records[index]
 
     @overload
-    def __getitem__(self, index: int) -> DatasetRecord:
-        ...
+    def __getitem__(self, index: int) -> DatasetRecord: ...
 
     @overload
-    def __getitem__(self, index: slice) -> List[DatasetRecord]:
-        ...
+    def __getitem__(self, index: slice) -> List[DatasetRecord]: ...
```

Comment:
I think black (formatter fyi) doesn't like this

Reply:
this was a result of black 😳

Reply:
ah im using 
```
black, 25.1.0 (compiled: yes)
Python (CPython) 3.13.4
```

i'll change the version and push up some changes hopefully 🤞

Reply:
i think that did it, thanks!